.template/spec/addons/base/phrase_app/template.rb

```diff
@@ -0,0 +1,9 @@
+describe 'PhraseApp addon - template' do
+  it 'creates .phraseapp.yml file' do
+    expect(file('.phraseapp.yml')).to exist
+  end
+  
+  it 'addings PhraseApp to codebase_spec' do
```

Comment:
`adds` seem more correct than `addings`

```suggestion
  it 'adds PhraseApp to codebase_spec' do
```